packages/backend/src/core/chart/core.ts

```diff
@@ -144,7 +144,8 @@ export default abstract class Chart<T extends Schema> {
 				columns[UNIQUE_TEMP_COLUMN_PREFIX + name] = {
 					type: 'varchar',
 					array: true,
-					default: '{}',
+					// REVIEW
+					default: () => 'array[]::varchar[]',
```

Comment:
`pg-mem` doesn't seem to map the default value `{}` correctly. However, I'm not 100 % sure if `array[]::varchar[]` is correct either, so we need to make sure that this change doesn't break anything on the real database.